### ISSUE TITLE
TypeError: metaclass conflict: the metaclass of a derived class must be a (non-strict) subclass of the metaclasses of all its bases

### ISSUE THREAD

**Issue by lekhnath** [absent1706/sqlalchemy-mixins#103]
Cannot use with Flask-SQLAlchemy

```
from flask_sqlalchemy import SQLAlchemy
from sqlalchemy_mixins import AllFeaturesMixin

db = SQLAlchemy()

class BaseModel(db.Model, AllFeaturesMixin):
    __abstract__ = True

```

I have created base model like above but getting this error:

```
TypeError: metaclass conflict: the metaclass of a derived class must be a (non-strict) subclass of the metaclasses of all its bases
```

How to fix this?

**NOTE: I've lazily initialized db in another file using db.initi_app method**

**Comment by lekhnath** [comment]
The usage example from readme also doesnot work.

https://github.com/absent1706/sqlalchemy-mixins#usage-with-flask-sqlalchemy

**Comment by michaelbukachi** [comment]
@lekhnath If you want to use it with SQLAlchemy, you have to use sqlalchemy-mixins < 2.0. Work is still being done to support it.

**Comment by michaelbukachi** [comment]
Hi @lekhnath. This should be resolved #108. Do you mind testing? 
Oversight on my part. Apologies.

**Comment by lekhnath** [comment]
I am getting following exception with `v2.0.3` :

```
sqlalchemy.exc.ArgumentError: Class '<class 'core.auth.models.group.AuthGroup'>' already has a primary mapper defined. 
```
